spec/unit/plugin_manager/offline_plugin_packager_spec.rb

```diff
@@ -67,7 +70,8 @@ def retrieve_dependencies_gems(path)
     before do
       FileUtils.mkdir_p(temporary_dir)
 
-      subject.package(plugins_args, target)
+      Stud.try(retries_count.times, retries_exceptions)  { subject.package(plugins_args, target) }
```

Comment:
I suggest adding a comment here mentioning that because the `package` method reaches the internet it's possible to see an IOError, so we've added a retry block.